app/src/Containers/EventSearchContainer.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+import { ScrollView, SafeAreaView, Button, Text } from 'react-native'
```

Comment:
I think `Button` and `Text` can be removed as they're not being used in this file?